build/attrs.csv

```diff
@@ -166,3 +166,12 @@ MaxVio,dbl,model
 OpenNodeCount,dbl,model
 PoolIgnore,int,var
 Work,dbl,model
+FuncPieceError,dbl,gconstr
+FuncPieceLength,dbl,gconstr
+FuncPieceRatio,dbl,gconstr
+FuncPieces,int,gconstr
+FuncNonlinear,int,gconstr
+GenConstrType,int,gconstr
```

Comment:
This is what it says in the reference:
> The language APIs contain enums that allows you to map the integer constraint type to a more meaningful name. In C, you'll find GRB_GENCONSTR_MAX, GRB_GENCONSTR_MIN, etc.

A quick search didn't yield any results for such an enum. Do you have any idea where it could be?

Reply:
I'll go with the order described in https://www.gurobi.com/documentation/11.0/refman/general_constraints.html for now.

I see that a distinction between "simple constraints" and "function constraints" is made in the article. I'm pretty sure that this isn't reflected in the enum representation (since you can't have data in enums in C and therefore no sub-enums). But maybe it would make sense to make the distinction in Rust? Something like this:

``` rust
#[non_exhaustive]
#[repr(i32)]
#[derive(Copy, Clone, Eq, PartialEq, Hash)]
pub enum GenConstrType {
    Simple(SimpleGenConstrType),
    Function(FunctionGenConstrType)
}

pub enum SimpleGenConstrType {
    Min = 0,
    Max = 1,
    ...
    Pwl = 7
}

pub enum FunctionGenConstrType {
    Polynomial = 8 /* would this be needed for `repr(i32)`? */
    ...
}
```
Now that I think of it, maybe not. But let me know what you think

Reply:
(partially?) addressed in 06c902dd2c0279418dc068977a2ab868cdc924d7